diff -r 4842bae3350f ns-3.17/src/wifi/model/dcf-manager.cc
--- a/ns-3.17/src/wifi/model/dcf-manager.cc	Fri Jun 07 15:31:52 2013 -0400
+++ b/ns-3.17/src/wifi/model/dcf-manager.cc	Mon Sep 09 16:05:32 2013 -0400
@@ -415,7 +415,8 @@
    * by notifying the collision to the user.
    */
   if (state->GetBackoffSlots () == 0
-      && IsBusy ())
+      && //IsBusy ())
+		GetBackoffStartFor (state) > Simulator::Now ())
     {
       MY_DEBUG ("medium is busy: collision");
       /* someone else has accessed the medium.
